### DISCUSSION TITLE
LS commercials rotations

### DISCUSSION THREAD

**Discussion by KjayCopper** [AzuraCast/AzuraCast#6349] [Q&A]
Hey guys I'm new to using Azura cast and i apologise if this question has been answered before, I couldn't see an answer specific like this 

I am looking to set up a playlist of commercials every 30 minutes of an hour
(ie 00:15 and 00:45 every hour)

However i can only see the option to set it up so it plays one song from the playlist then goes back to music
is there a way that i can do it so it plays 3 commercials per 30 minutes?

Example:

00:00 - plays Top of hour jingle
00:15 - Plays X commercials from playlist
00:30 - Random radio jingle
00:45 - Plays X commercials from playlist

(X being the specified amount)

Thanks in advanced

**Comment by LordHelmchen666** [comment]
I do it this way.
![Screenshot_20230615_152605](https://github.com/AzuraCast/AzuraCast/assets/43477041/5b6ac23f-c412-4f5e-8bcb-1d623cf0fd28)
![Screenshot_20230615_152702](https://github.com/AzuraCast/AzuraCast/assets/43477041/d6809ffe-c82b-4c47-9957-e5ae0c987e75)
![Screenshot_20230615_152732](https://github.com/AzuraCast/AzuraCast/assets/43477041/f8348db3-16c4-4633-b228-fa16a30f2fac)
![Screenshot_20230615_152911](https://github.com/AzuraCast/AzuraCast/assets/43477041/fbdd5f6d-c8cc-4761-9d28-6e2d6c28ae3e)

But sometimes this playlists is  getting in a loop. So you have to monitor your stream! I am searching what causes this issue. I believe it happens when you have short tracks, e.g. 3 seconds, at the start or end of the playlist. Right now i have tracks with minimum 17 seconds and it seems not to go in a loop. Working for seven days now, but I'll need to monitor it for a longer time.
